hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/submerkle/ExpirableTxnRecord.java

```diff
@@ -1000,6 +1000,10 @@ public boolean shouldNotBeExternalized() {
             return onlyExternalizedIfSuccessful && !TxnReceipt.SUCCESS_LITERAL.equals(receiptBuilder.getStatus());
         }
 
+        public boolean isPendingSuccess() {
+            return TxnReceipt.SUCCESS_LITERAL.equals(receiptBuilder.getStatus());
```

Comment:
Do we use `.equals()` for enum values (instead of `==`)?  (They don't over in the mirror node repo.)

Reply:
`SUCCESS_LITERAL` is a string.